pyfr/mputil.py

```diff
@@ -25,6 +25,16 @@ def jacobi(n, a, b, z):
     return j
 
 
+def derivative(f, x):
```

Comment:
Why do we have this function?  mpmath already provides numerical differentiation routines and, moreover, why do we not have a `jacobi_diff` method which computes the first differential of the Jacobi polynomials analytically?